app/uk/gov/hmrc/emailverification/ErrorHandler.scala

```diff
@@ -29,38 +29,40 @@ import uk.gov.hmrc.play.HeaderCarrierConverter
 import uk.gov.hmrc.play.audit.http.connector.AuditConnector
 import uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler
 import play.api.libs.concurrent.Execution.Implicits._
+import uk.gov.hmrc.play.bootstrap.config.HttpAuditEvent
 
 import scala.concurrent.Future
 
-class ErrorHandler @Inject()(configuration: Configuration, auditConnector: AuditConnector) extends JsonErrorHandler(configuration,auditConnector){
+class ErrorHandler @Inject()(configuration: Configuration, auditConnector: AuditConnector, httpAuditEvent: HttpAuditEvent) extends JsonErrorHandler(auditConnector, httpAuditEvent, configuration) {
```

Comment:
this class looks very similar to the built-in bootstrap-play `JsonErrorHandler` - we could possibly replace this class with that